### ISSUE TITLE
Improve accessibility by fixing issues reported by the Accessibility Fundation

### ISSUE THREAD

**Issue by stephanegigandet** [openfoodfacts/openfoodfacts-server#4640]
The Accessibility Foundation has made an accessibility of the Open Food Facts web site. There is a number of issues that they pointed out and that we should fix.

[Report OpenFoodFact 15-12-2020.pdf](https://github.com/openfoodfacts/openfoodfacts-server/files/5702582/Report.OpenFoodFact.15-12-2020.pdf)

Who would be interested in fixing some of them?

**Comment by jolesh** [comment]
Is there a good way of keeping track of what we already have implemented and what is still to do? A project for example?

**Comment by Ban3** [comment]
> Is there a good way of keeping track of what we already have implemented and what is still to do? A project for example?
Not yet, but doing something like that was my first thought when reading the report. A project sounds like a good idea.

**Comment by stephanegigandet** [comment]
@ban3 @jolesh : a project is a good idea, does one of you want to create it, and create the corresponding bugs with the items mentioned in the report?

@leventgunay : do you still want to address some of the reported issues to improve accessibility?

**Comment by roshnaeem** [comment]
@stephanegigandet can I start working on it? I want to address few issues to improve accessibility.
.

**Comment by stephanegigandet** [comment]
Hi @roshnaeem , sure, thank you!
Could you create a separate bug for each of the issues you intend to work on, and then link them from this bug?
That way it will be easier to discuss each issue.
I just created one for the links underlining: https://github.com/openfoodfacts/openfoodfacts-server/issues/5074

**Comment by roshnaeem** [comment]
Yes, sure @stephanegigandet. It will be easier for everyone.

**Comment by yuktea** [comment]
> @Ban3 @jolesh : a project is a good idea, does one of you want to create it, and create the corresponding bugs with the items mentioned in the report?
> 
> @leventgunay : do you still want to address some of the reported issues to improve accessibility?

@stephanegigandet should I work on creating this project? (I'll point out and document what we have fixed and create issues for what we have not from the accessibility report)

**Comment by alexgarel** [comment]
@yuktea yes, go ahead, it's a good idea. You should verify for some points are already fixed.
